aspnet/mvc/views/forms/sample/final/ViewModels/DescriptionViewModel.cs

```diff
@@ -0,0 +1,7 @@
+﻿using System.ComponentModel.DataAnnotations;
+public class DescriptionViewModel
+{
+    [MinLength(5), MaxLength(1024)]
```

Comment:
This works but looks strange compared to `[StringLength(1024, MinimumLength = 5)]` for a `string` property. At least split the two attributes to make this more readable e.g.
``` c#
[MinLength(5)]
[MaxLength(1024)]
```